test/image-diff_test.js

```diff
@@ -45,12 +45,51 @@ describe('image-diff', function () {
     });
   });
 
+  describe('diffing different images using lower threshold', function () {
```

Comment:
Due to my confusion, I had this test case wrong. It should be named something like:

```
comparing different images over a threshold
```